tests/test_lambda.py

```diff
@@ -676,3 +676,43 @@ def test_cel_lambda_related_subnets(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
         self.assertEqual(resources[0]['c7n:matched-subnets'], ['subnet-07c118e47bb84cee7'])
+
+    def test_cel_lambda_cross_account(self):
+        session_factory = self.replay_flight_data("test_lambda_permission_matched")
+        client = session_factory().client("lambda")
+        name = "func-b"
+
+        self.create_function(client, name)
```

Comment:
Don't want to create resources in unit test. Custodian used to use this, but has since moved away from it to simplify unit tests